google/cloud/dataproc_spark_connect/session.py

```diff
@@ -407,9 +409,29 @@ def _get_dataproc_config(self):
                     )
                 }
             if "COLAB_NOTEBOOK_ID" in os.environ:
-                dataproc_config.labels["colab-notebook-id"] = os.environ[
-                    "COLAB_NOTEBOOK_ID"
-                ]
+                colab_notebook_id_full = os.environ["COLAB_NOTEBOOK_ID"]
+                # Expected format: /embedded/projects/proj-id/locations/us-central1/repositories/notebook-id
```

Comment:
Do we know whether notebook ID is always has this labels-compliant ID? Can it be in different case, have different characters.